index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/7bfe064e41.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portofolio Page</title>
+</head>
+<body>
+<header>
+  <h1 class="log"><a href="#">Ndonyi</a></h1>
+  <nav>
+    <div class="bar"></div>
+    <div class="bar"></div>
+    <div class="bar"></div>
+  </nav>
+</header>
+<section class="container">
+<section class="header-text">
+    <h1>I’m Ndonyi <br>
+        Glad to see <br>you!</h1>
+  <p>Hello I’m a software developer!
+     I can help you build a product , feature 
+     or website Look through some of my work and 
+     experience! If you like what you see and have a
+      project you need coded,
+     don’t hestiate to contact me.</p>
+</section>
+
+<section class="contact_list">
+  <h3 >LET’S CONNECT</h3>
+    <ul>
+        <li><a href="https://twitter.com/home"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="https://www.linkedin.com/feed/?trk=homepage-basic_google-one-tap-submit"><i class="fa-brands fa-linkedin-in"></i></a></li>
+        <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+        <li><a href="https://github.com/citec-47"><i class="fa-brands fa-github"></i></a></li>
+        <li><a href="https://www.facebook.com/ndonyi.maurice"></i></a></li>
+     </ul>
+    </section>
+</section>
+<section>
+<div class="project">
+    <div class="project_1st">
+        <div class="four">
+            <img src="./Images/IMAGE5.PNG.png" alt="">
+        </div>
+      <div class="detail">
+        <h2>Tonic</h2>
+        <ul>
+            <li><a href="#">CANOPY</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">2015</a></li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="all">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <br>
+        <br>
+          <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
+        </div>
+    </div>
+   
+  </div>
```

Comment:
Currently, your styles are not exactly lookalike to the Figme design template you have chosen. Remember your style needs to be like a template at least above 375px until it hits desktop size. Currently, they are not as it's flowing out of their container height. I am attaching a screenshot of your design to make it more clear. 

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/219420165-741670c1-cb7d-4dd8-8bb7-72ca90553994.png)